src/elli/importer/accurion.py

```diff
@@ -0,0 +1,99 @@
+"""A helper class to load data from Accurion EP4 DAT files.
+Typical files look like: Si3N4_on_4inBF33_W03_20240905-105631.ds.dat
+"""
+
+import pandas as pd
+import chardet
+import numpy as np
+
+def detect_encoding(fname: str):
+	r"""Detects the encoding of file fname. Used in read_accurion_psi_delta only.
+  Args:
+    fname (str): Filename
+  """
+	with open(fname, 'rb') as f:
+		raw_data = f.read()
+	result = chardet.detect(raw_data)
+	return result['encoding']
+
+def read_accurion_psi_delta(fname: str, wrap=True) -> pd.DataFrame:
+	r"""Read a psi/delta Accurion dat file.
+	
+  Args:
+    fname (str): Filename of the measured dat file
+    wrap (bool): Whether delta values should be wrapped; should always be True
```

Comment:
It's not entirely clear to me what this means. Also if it should always be `True` I would argue that we don't really need it as a parameter.